core/app/models/spree/shipping_rate.rb

```diff
@@ -7,39 +7,29 @@ class ShippingRate < Spree::Base
     delegate :order, :currency, to: :shipment
     delegate :name, to: :shipping_method
 
-    def display_base_price
-      Spree::Money.new(cost, currency: currency)
-    end
+    extend Spree::DisplayMoney
+
+    money_methods :base_price, :tax_amount
 
-    def calculate_tax_amount
-      tax_rate.calculator.compute_shipping_rate(self)
+    def base_price
+      cost
     end
 
     def display_price
       price = display_base_price.to_s
-      if tax_rate
-        tax_amount = calculate_tax_amount
-        if tax_amount != 0
-          if tax_rate.included_in_price?
-            if tax_amount > 0
-              amount = "#{display_tax_amount(tax_amount)} #{tax_rate.name}"
-              price += " (#{Spree.t(:incl)} #{amount})"
-            else
-              amount = "#{display_tax_amount(tax_amount*-1)} #{tax_rate.name}"
-              price += " (#{Spree.t(:excl)} #{amount})"
-            end
-          else
-            amount = "#{display_tax_amount(tax_amount)} #{tax_rate.name}"
-            price += " (+ #{amount})"
-          end
-        end
-      end
-      price
+
+      return price if tax_rate.nil? || tax_amount == 0
+
+      Spree.t tax_rate.included_in_price? ? :including_tax : :excluding_tax,
+              scope: 'shipping_rates.display_price',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.